### ISSUE TITLE
[low priority question] We have empty thumbnails for records that couldn't be thumbnailed. Are we okay with that? Other options?

### ISSUE THREAD

**Issue by christinklez** [ucldc/rikolti#938]
Here's an example of a record that couldn't be thumbnailed (grey tile with a white circle in the middle)
https://calisphere-stage.cdlib.org/item/ark:/21198/zz0025pngj/

And here's how it looks in a search result (appears as a grey tile)
https://calisphere-stage.cdlib.org/collections/28111/?q=&sort=a&type_ss=&rq=Ethel%20Sissle%2C%20Catherine%20Garcia%20and%20Marva%20Louis%2C%20Los%20Angeles%2C%201940s

We're here because (typically) there is an issue with the source thumbnail. There have been some updates to the content harvester to push through the run, so that these "errors" don't block harvesting. And thank you for doing that!

We also initially had broken-image icons appearing in this place, before we started getting these grey tiles. To be clear, these grey tiles are a huge improvement from where we started. Example: https://calisphere-stage.cdlib.org/item/ark:/28722/bk001541x0r/ -- note that a re-run from content harvesting will now produce a grey tile instead.

A low priority question: Are there options to include last resort type icons here?

(Side note: Since these issues are typically originating on the data provider end, we should report these to contributors.)

**Comment by christinklez** [comment]
Discussed that these are likely items that do not have a `type` value provided, or they may but it may not be in our expected vocabulary.
